src/components/Select/Select.component.tsx

```diff
@@ -132,17 +162,18 @@ export const Select = (props: SelectProps) => {
 		<Wrapper>
 			<Input
 				ref={inputRef}
-				placeholder=''
+				placeholder=' '
```

Comment:
why this change?

Reply:
In the select component, we are checking if the placeholder is empty of not
if the placeholder is empty, then the label will align b/w border else inside input